The plugin hasn't been tested with an upcoming version of WordPress

There is an upcoming WordPress version in the **release candidate** stage that the plugin hasn't been tested with. Please test it and then change the "Tested up to" field in the plugin readme.

**Tested up to:** 6.7
**Upcoming version:** 6.8

This issue will be closed automatically when the versions match.

@jeffpaul Successfully tested uploading SVGs to media libraryin #251 - yet there is one test failing:

  Admin can add SVG block to a post:
     AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Blocks' within the element: <button#tabs-0-edit-post/block.css-8ivkzm.enfox0g3> but never did.

The SafeSVG block is not registering. Needs investigation.

--- a/readme.txt
+++ b/readme.txt
@@ -1,7 +1,7 @@
 === Safe SVG ===
 Contributors:      10up, enshrined, jeffpaul
 Tags:              svg, security, media, vector, mime
-Tested up to:      6.7
+Tested up to:      6.8
 Stable tag:        2.3.1
 License:           GPL-2.0-or-later
 License URI:       https://spdx.org/licenses/GPL-2.0-or-later.html

